src/Sarif.Viewer.VisualStudio/Views/ResolveEmbeddedFileDialog.xaml.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Controls;
+
+using Microsoft.VisualStudio.PlatformUI;
+
+namespace Microsoft.Sarif.Viewer.Views
+{
+    /// <summary>
+    /// Interaction logic for ResolveEmbeddedFileDialog.xaml.
+    /// </summary>
+    public partial class ResolveEmbeddedFileDialog : DialogWindow
```

Comment:
>ResolveEmbeddedFileDialog [](start = 25, length = 25)

Check if this is IDisposable. If yes, unhook the onloadaed

Reply:
inherited from IDispose

---
In reply to: [601276773](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r601276773) [](ancestors = 601276773)